docs/EC-UI-screenshot.md

```diff
@@ -0,0 +1,148 @@
+# EC WEB-UI Screenshots as per Tabs
+
+
+## 1)View:
+
+1.1)View: Topology View Table
+
+![image](https://user-images.githubusercontent.com/76679290/109985830-30292280-7d2b-11eb-8dbc-78e256fed96b.png)
```

Comment:
please update the image link to the relative path. 

https://stackoverflow.com/questions/10189356/how-to-add-screenshot-to-readmes-in-github-repository

Reply:
@vidya22022 @Subhash-vishwakarma

Reply:
*image links